<?xml version='1.0' encoding='utf-8'?>
<sdf version="1.6">
  <world name="default">
    <!-- A Custom light source -->
    <light type="directional" name="light_source" cast_shadows="false">
      <origin pose="0 0 30 0 0 0"/>
      <diffuse rgba=".9 .9 .9 1"/>
      <specular rgba=".1 .1 .1 1"/>
      <attenuation range="20"/>
      <direction xyz="0 0 -1"/>
    </light>
    <!-- A ground plane -->
    <include>
      <uri>model://ground_plane</uri>
    </include>
    <!-- A gazebo links attacher -->
    <plugin name="ros_link_attacher_plugin" filename="libgazebo_ros_link_attacher.so"/>
    <!-- Declaring Physics in Gazebo-->
    <gravity>0 0 -9.8</gravity>
    <physics default="0" name="default_physics" type="ode">
      <max_step_size>0.001</max_step_size>
      <real_time_factor>1</real_time_factor>
      <real_time_update_rate>1000</real_time_update_rate>
    </physics>
    <!-- Importing a 2D Camera in Gazebo -->
    <model name="camera_1">
      <include>
        <uri>model://camera1</uri>
        <pose>0 0 4.6 1.57 1.57 0</pose>
      </include>
      <static>true</static>
    </model>
    <!-- Importing the arena in Gazebo -->
    <model name="arena">
      <include>
        <uri>model://arena_s2</uri>
        <static>true</static>
      </include>
      <pose frame="">0 0 0.01 0 0 0</pose>
      <static>true</static>
    </model>

    <gui fullscreen='0'>
      <camera name='user_camera'>
        <pose frame=''>-2.75 4.19 5.15 0.0 0.72 -0.99</pose>
        <view_controller>orbit</view_controller>
        <projection_type>perspective</projection_type>
      </camera>
    </gui>

  </world>
</sdf>